### PR TITLE
Remove IP hardcoding

### PR DESCRIPTION
**Pull request by prabodh1194**
Fix for this issue #1

### CODE REVIEW

File: pcap_to_ditg/pcap_to_ditg.py [hunk 1]
```diff
@@ -213,10 +199,11 @@ def __generateMapper(self):
             if os.path.exists(self.list_file):
                 for i in range(int(endPoints[0]), int(endPoints[1]) + 1):
                     ip = linecache.getline(self.list_file, i).strip().strip(',')
-                    (self.__IpMapDict)[ip] = pcap_to_ditg.__getIpForHost(p)
+                    (self.__IpMapDict)[ip] = endPoints[2]
             else:
                 print('*** The file \'' + self.list_file + '\' could not be read\n')
                 raise IOError('The file \'' + self.list_file + '\' could not be read\n')
+        print self.__IpMapDict
```
Comment (devenbansod) [inline pcap_to_ditg/pcap_to_ditg.py]: Stray debugging statement ?

**Review by devenbansod — CHANGES_REQUESTED** [review]
Just squash the commits. 
Otherwise it looks good.

**Comment by prabodh1194** [comment]
crap, squashing made it uglier.

**Comment by devenbansod** [comment]
* `git rebase -i HEAD~4`
* Change 2nd, 3rd and 4th commit from `pick` to `s`
* Save and Close the file
* Edit the commit message (editor should auto-open). Save it and close it.
* Push force using  `git push -f <remote> <branch>`

**Comment by prabodh1194** [comment]
Done. Thanks.

**Review by devenbansod — APPROVED** [review]

**Comment by devenbansod** [comment]
Great! 
Merged! Thanks!

**Pull request merged**